<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nickname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>닉네임</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_birth">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>생년월일</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>157</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>완료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>187</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
